src/SDK/Common/Export/Http/PsrTransport.php

```diff
@@ -110,7 +110,7 @@ public function send(string $payload, ?CancellationInterface $cancellation = nul
                 return new ErrorFuture($e);
             }
 
-            if ($retries + 1 === $this->maxRetries) {
+            if ($retries === $this->maxRetries) {
```

Comment:
what happens if retries is negative? perhaps this should be a `>=` test...

Reply:
Tempted to say "you get what you deserve" if you pass a negative maxRetries 😂 
I've pushed up a change so it's `>=` now.